fastar/rules.py

```diff
@@ -188,17 +174,17 @@ def slice_dependency_rule(
   out_shape = np.asarray(outcount.shape)
   if strides is None:
     inbox = np.add(start_indices, outstart), out_shape
-    return [inbox], [np.ones(inbox[1], int)], (None, None)
+    return [inbox], [np.ones(inbox[1], int)], (dict(strides=tuple([1] * operand.ndim)), None)
```

Comment:
`operand.ndim * (1,)` is shorter than
`tuple([1] * operand.ndim)`

Reply:
Good point! (Weird, somehow I thought this only works for lists.)